internal/flickr/handlers/get_restlist_read.go

```diff
@@ -0,0 +1,134 @@
+package handlers
+
+import (
+	"context"
+
+	util "github.com/anz-bank/sysl-template/internal"
+	"github.com/anz-bank/sysl-template/internal/gen/flickr"
+)
+
+// GetRestListRead reads photos from flickr
```

Comment:
```suggestion
// GetRestListRead reads photos from Flickr.
```

Reply:
The go comments normally don't use a snake case in between a comment.